FreeTAKServer/controllers/configuration_wizard.py

```diff
@@ -5,6 +5,10 @@
 yaml = YAML()
```

Comment:
It does, but I want to leave that as another task rather than add more changes to this PR. I have a feeling that the config wizard will change dramatically and really just generate a YAML file for the user to use. I have not fully thought this out yet, so I reserve the right to change my mind :smile: